docs/api/blueprint/session/favourite_sessions.apib

```diff
@@ -0,0 +1,179 @@
+# Group Favourite Sessions
+
+This Group's APIs can be used for adding a particular session to the favourite list of the user.
+
+## Favourite Sessions Collection [/v1/user-favourite-sessions]
```

Comment:
This should not be allowed, there is not use case for this

These are required list endpoints

/user/ID/favourite-sessions -> All favourite sessions of a user, only queriable by the user itself

/event/ID/favourite-sessions -> All favourite sessions of currently logged in user in an event

/event/ID/favourite-sessions?all=true -> All favourite sessions in an event regardless of user, only queryable by co-organizer. Can be implemented later

/session/ID/favourit... -> All who favourited this session

Reply:
@iamareebjamal the detail and delete endpoints are also required right?

Reply:
Yes, the above comment is only for list endpoints